packages/oruga/src/components/datepicker/Datepicker.vue

```diff
@@ -854,11 +854,11 @@ function onChange(value: string): void {
     const date = (props.dateParser as any)(value, defaultDateParser);
```

Comment:
Could you here add a specific type definition `const date: Date = ` please :)

Reply:
At this point, `date` might not be a `Date`, though; it could also be a `[Date, Date]`  (at least if `defaultDateParser` handled ranges correctly) or `null`. It seems better to me to do an explicit type check.